src/coreclr/jit/valuenum.cpp

```diff
@@ -454,7 +454,7 @@ ValueNumStore::ValueNumStore(Compiler* comp, CompAllocator alloc)
     // We have no current allocation chunks.
     for (unsigned i = 0; i < TYP_COUNT; i++)
     {
-        for (unsigned j = CEA_None; j <= CEA_Count + BasicBlock::MAX_LOOP_NUM; j++)
+        for (unsigned j = CEA_Const; j <= CEA_Count; j++)
```

Comment:
Is this change a bug-fix, as we weren't fully initializing things?

Reply:
I see now that you are just removing the MAX_LOOP_NUM stuff.
You did change the start of this loop from CEA_None to CSE_Const

Reply:
Yep, we no longer need the `CEA_None` chunks.